static/src/javascripts/lib/dom.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+
+export const wrappedElementsFromString = (
```

Comment:
@guardian/dotcom-platform

Having stripped out most of the usages of `$` in `acquisitions-epic-liveblog.js` without any cost, it occurred to me that in the only things that were significantly more verbose without `$` were `$.create()` and `.insertAfter()`

So I wondered whether in general it might be best to have a handful of our own library functions (which we can then add types to) rather than the bonzo/bean/qwery combination, none of which have been maintained for years and which have been made largely redundant by advances in the native DOM APIs plus polyfill.io